mindocr/models/backbones/blocks/magca.py

```diff
@@ -0,0 +1,155 @@
+import mindspore.nn as nn
+import mindspore.ops as ops
+from mindspore import Tensor
+
+__all__ = ["MultiAspectGCAttention"]
```

Comment:
`__all__` is not necessary since classes with names starting with underscore will not be exported (`_LayerNorm`).

Reply:
`ops` and `Tensor` will also be imported if we do not add `__all__`, and try to `import * from magca`, which is not we expected